.codecov.yml

```diff
@@ -7,6 +7,6 @@ coverage:
   ignore:
      - "tests/.*"
      - "examples/.*"
-     - "src/metatensor/models/experimental/.*"
+     - "src/metatrain/experimental/.*"
```

Comment:
Do we want `src/metatrain` or just `metatrain`? Isn't `src` redundant since everything is inside `src/metatrain/`?

Reply:
This is what setuptools recommends: https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#src-layout